testsuite/simulation/modelica/initialization/homotopy6.mos

```diff
@@ -0,0 +1,60 @@
+// name: homotopy6
+// keywords: initialization, homotopy
+// status: correct
+// cflags:
+// teardown_command: rm -rf initializationTests.homotopy5* _initializationTests.homotopy5* output.log
+// cflags: -d=-newInst
+//
+//  Check if equation system will get a different solution for each global lambda value.
```

Comment:
```suggestion
// Check if equation system actually goes through all steps in the global homotopy path.
// To prevent regressions like https://github.com/OpenModelica/OpenModelica/issues/8097
```